tests/spread/plugins/catkin/snaps/fake-ros-package-kyrofa/snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: fake-ros-package-kyrofa
```

Comment:
can you rename this to `test-snapcraft-fake-ros-package-core18`? Or will this work with other bases?

Reply:
It will not work with other bases given that "melodic" is hard-coded (which snapcraft will only use on core18). I've renamed it.